packages/react-components/src/components/VideoTile.tsx

```diff
@@ -77,20 +77,31 @@ export interface VideoTileProps {
    * @remarks `displayName` is used to generate avatar initials if `initialsName` is not provided.
    */
   displayName?: string;
-  /** Name of the participant used to generate initials. For example, a name `John Doe` will display `JD` as initials.
+  /**
+   * Name of the participant used to generate initials. For example, a name `John Doe` will display `JD` as initials.
    * @remarks `displayName` is used if this property is not specified.
    */
   initialsName?: string;
+  /**
+   * Minimum size of the personal avatar.
```

Comment:
What unit is this in?

Should we include the unit in the name of the field (since we don't have it in the type). So `personaMinSizePx`.

[...some time later...]

Phew... jumped through 5 components to understand exactly what UI element this sizes. It's not obvious at all from the props here what the effect of setting it will be. We should add more details to docstring to make this usable.

"Sets the minimum size in pixel of the Persona coin shown when no video stream is available"

tbh, even that isn't clear to me (what is "size of a coin"? radius?, but `AvatarPersona` doesn't do a good job of documenting this either).

Reply:
updated docstring